marlowe-dashboard-client/src/Play/State.purs

```diff
@@ -173,6 +175,43 @@ handleAction _ CloseCard = do
   for_ (init cards) \remainingCards ->
     assign _cards remainingCards
 
+-- FIXME: until the PAB is working, we get the latest data from local storage here (this action
+-- is called every second)
+handleAction { currentSlot } UpdateFromStorage = do
```

Comment:
I think it's better to add a subscription to local storage events rather than polling

Reply:
yep, that's my plan for today - this is an outdated comment from when I was exprimenting with polling (it didn't work well :upside_down_face:)